SessionMessagingKit/Open Groups/OpenGroupManager.swift

```diff
@@ -529,11 +533,17 @@ public final class OpenGroupManager: NSObject {
             .filter { $0.deleted == true }
             .map { $0.id }
         
-        // Update the 'openGroupSequenceNumber' value (Note: SOGS V4 uses the 'seqNo' instead of the 'serverId')
         if let seqNo: Int64 = seqNo {
+            // Update the 'openGroupSequenceNumber' value (Note: SOGS V4 uses the 'seqNo' instead of the 'serverId')
             _ = try? OpenGroup
                 .filter(id: openGroup.id)
                 .updateAll(db, OpenGroup.Columns.sequenceNumber.set(to: seqNo))
+            
+            // Update pendingChange cache
+            dependencies.mutableCache.mutate {
+                $0.pendingChanges = $0.pendingChanges
+                    .filter { $0.seqNo == nil || $0.seqNo! > seqNo }
```

Comment:
Do we want to remove entries with no `seqNo` here? (that would include any in-flight changes)

Reply:
Hmm... I'm keeping those entries with no `seqNo` or `seqNo` greater than current polling's `seqNo`

Reply:
Oh sorry I misread this - you are right this code is correct 😅